cernopendata/modules/fixtures/data/docs/lhcb-masterclass/lhcb-masterclass.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "body": {
+      "content": "lhcb-masterclass.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "experiment": "LHCb"
+      }
+    ],
+    "experiment": "LHCb",
+    "short_description": {
+      "content": "Instructions on the LHCb Masterclass exercise are provided here"
+    },
+    "slug": "lhcb-masterclass",
```

Comment:
The new `/docs/lhcb-masterclass` page is not linked from anywhere. It may be good to link it from either

- https://opendata.cern.ch/docs/about-lhcb
- https://opendata.cern.ch/docs/lhcb-getting-started

Reply:
Yes, the masterclass is actually a stand-alone quick exercise for school students. It doesn't really use LHCb open data, only a processed sample via some web-applet. It could be quite confusing so maybe it's not even a very bad idea not to link it directly. This is only used by some institutes when they are holding these masterclass sessions.